products/qt.md

```diff
@@ -44,7 +44,7 @@ releases:
 -   releaseCycle: "5.15"
     cycleShortHand: 515
     eol: 2023-05-26
-    latest: "5.15.2"
+    latest: "5.15.10"
```

Comment:
Any link for 5.15.10?

I only see 5.15.4 here: https://github.com/qt/qt5/tags

Reply:
Closest would be this I believe due to it being commercial only: https://www.qt.io/blog/commercial-lts-qt-5.15.10-released

Reply:
@captn3m0 as like @Evernow mentioned 5.15.10 released but because of it is commerial license owners on the other hand
KDE is keeping applied all patches and release them on its git repos.

Reply:
Really need to make this clearer on the page itself.